test/ably/restrequest_test.py

```diff
@@ -1,6 +1,6 @@
-import time
+#import time
```

Comment:
Mind fi I ask why this is commented out?

Reply:
Because it is not used, so pyflakes complains. I can remove it instead of commenting. I could also add a commit to make pyflakes happy in every other file.

Reply:
I prefer that if code is not used it's removed as opposed to commented out. We can always get code back through git history.